README.md

```diff
@@ -12,6 +13,17 @@ This is a prototype debugger written without any XUL and based on React and Redu
 
 ![screen shot 2016-05-16 at 1 24 29 pm](https://cloud.githubusercontent.com/assets/254562/15297643/34575ca6-1b69-11e6-9703-8ba0a029d4f9.png)
 
+----
+#### Getting Involved
+
+The Debugger is an open source project and we would love your help. We have prepared a [contributing](https://github.com/jlongster/debugger.html/blob/master/CONTRIBUTING.md) guide to help answer some of the common questions.
+
+Also, please feel free to stop by and say hi! We're all in Mozilla's [Devtools](irc://irc.mozilla.org/devtools) channel. If you do not have IRC, we recommend using the free version of [irccloud](https://www.irccloud.com/). 
++ Address `irc.mozilla.org`
++ Port `6697`
++ Channel `devtools`
+
+----
```

Comment:
very nice. It might be a little more condensed to just say irc.mozilla.org channel #devtools? The port is the default port, so I don't think you really need to specify it.